rs/replicated_state/src/canister_snapshots.rs

```diff
@@ -284,6 +284,10 @@ pub struct ExecutionStateSnapshot {
     /// The raw canister module.
     #[validate_eq(Ignore)]
     pub wasm_binary: CanisterModule,
+    /// The Wasm global variables.
+    /// Note: The hypervisor instrumentations exports all global variables,
+    /// including originally internal global variables.
+    pub exported_globals: Vec<Global>,
```

Comment:
```suggestion
    #[validate_eq(Ignore)]
    pub exported_globals: Vec<Global>,
```

This field should be treated the same as the corresponding field in the canister for the purpose of validation during checkpoints.

Reply:
Thank you, I adjusted the code analogous to the canister state. I assume it is okay to skip this during checkpoint validation?

Reply:
Yeah it's one of those fields where we weren't sure if it's expensive to compare, so we set it to ignore for now. We might change that in the future, but either way canister states and snapshots should treat it the same.